mantle/cmd/kola/options.go

```diff
@@ -231,5 +231,13 @@ func syncCosaOptions() error {
 		}
 	}
 
+	if kola.Options.Distribution == "" && kola.CosaBuild != nil {
```

Comment:
Hmm, I think in `syncCosaOptions`, we can assume that `kola.CosaBuild` is provided.